### ISSUE TITLE
Add label or title element to input class

### ISSUE THREAD

**Issue by tpcolson** [nationalparkservice/npmap-species#103]
Add label or title (or both?) element to input class

`<input class='dropdown-input' oninput='fuseSearch(0, this.value);'>`

so the control function is passed to a screen reader. In addition, each control and/or user-selectable item should have a label and/or title attribute.

**Comment by jduggan1** [comment]
Done with https://github.com/nationalparkservice/npmap-species/commit/28ca1782e26c27463db50e849579a572eef2d861
